README.md

```diff
@@ -101,7 +102,7 @@ After you're finished, make sure to call `te_free()`.
 
     int err;
     /* Compile the expression with variables. */
-    te_expr *expr = te_compile("sqrt(x^2+y^2)", vars, 2, &err);
+    te_expr *expr = te_compile("sqrt(x**2+y**2)", vars, 2, &err);
```

Comment:
Why the change to `**` from `^`? I don't necessarily oppose it, using Python frequently myself, but it's certainly not a mathematical standard practice (Mathematica, Matlab).